### ISSUE TITLE
Get cardinality info from XML schema

### ISSUE THREAD

**Issue by Bjwebb** [OpenDataServices/flatten-tool#177]

**Comment by stevieflow** [comment]
I guess this is further compounded by the instance where there could be multi-nested elements, such as iati-activity/result/indicator : https://discuss.iatistandard.org/t/challenge-results-with-multiple-indicators-and-multiple-measures/1108/

Is this the same issue?

**Comment by Bjwebb** [comment]
No, I think that's a slightly different issue.

**Comment by Bjwebb** [comment]
I've done some work to at least make this consistent within a particular file. ie. `transaction` will have its own sheet if there is more than one transaction for any activity in a file, otherwise it will appear on the `iati-activity` sheet.

Dev branch: `177-xml-path-consistency`
CoVE demo link: http://flattentool-177-xml-path-consistency.dev.cove.opendataservices.coop/

**Comment by stevieflow** [comment]
Thanks @Bjwebb - I can see that working through conversion of the sample file, as that has more than one transaction.  I can confirm that these are all then converted through to their own sheet

**Comment by stevieflow** [comment]
I noticed something via this activity

http://d-portal.org/q.xml?aid=US-GOV-1-SD-AID-FFP-G-16-00035

In each transaction there are two sectors, but from different vocabularies (which is in keeping with the standard rules, it seems)

If you flatten this, the transaction sectors are pushed to a new sheet.  But - given that this particularly activity has multiple transactions, there's then no way to understand which sectors relate back to which transactions....

**Comment by Bjwebb** [comment]
@stevieflow That's right, I've made a new issue for that https://github.com/OpenDataServices/flatten-tool/issues/295
